content/systems/filecoin_markets/onchain_storage_market/storage_deal_states.md

```diff
@@ -30,10 +30,14 @@ The following describes how a deal transitions between its different states.
   - The sector containing the deal has expired. This is triggered by `StorageMinerActorCode._submitPowerReport` which is called whenver a PoSt is submitted. Power associated with the deals in the sector will be lost, collaterals returned, and all remaining storage fees unlocked.
   - The sector containing the active deal has been terminated. This is triggered by `StorageMinerActor._submitFaultReport` for `TerminatedFaults`. No storage deal collateral will be slashed on fault declaration or detection, only on termination. A terminated fault is triggered when a sector is in the `Failing` state for `MAX_CONSECUTIVE_FAULTS` consecutive proving periods.
 
-Given deal states and their transitions, the following are the relationships between deal states and other economic states and activities in the protocol.
+Given the **onchain deal states and their transitions** discussed above, below is a description of the relationships between **off-chain deal states** and other economic states and activities in the protocol. Note that these states below are **off-chain states tracked internally by the markets module**.
```

Comment:
This is incorrect. The economic activites listed: power, deal payment, deal collateral are connected to the *on chain deals states* not the off chain deal states. Please change to this and remove the sentence about off chain states.

```suggestion
Given the **onchain deal states and their transitions** discussed above, below is a description of the relationships between **onchain deal states and their transition** and other economic states and activities in the protocol. 
```